qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -104,6 +137,20 @@ public void test20PluginsListWithNoPlugins() {
         assertThat("Expected no plugins to be listed", r.stdout, emptyString());
     }
 
+    /**
+     * Check that the IngestUserAgentPlugin is enabled.
+     */
+    public void test21IngestUserAgentPluginIsInstalled() throws Exception {
```

Comment:
I'm not sure these checks are useful anymore. While it user agent and geoip plugins used to only be bundled in docker, we now have these in the oss/default distributions, and have appropriate checks already. It is duplication like this that worries me on the docker tests becoming copies of every other packaging test we have, and why I think most of this file needs to be deleted and instead the similar tests extracted out to work across distributions, regardless of whether that distro is docker based or not.